lib/librato/rack/configuration.rb

```diff
@@ -55,6 +55,7 @@ def explicit_source?
       def load_configuration
         self.user = ENV['LIBRATO_USER'] || ENV['LIBRATO_METRICS_USER']
         self.token = ENV['LIBRATO_TOKEN'] || ENV['LIBRATO_METRICS_TOKEN']
+        self.proxy = ENV['LIBRATO_PROXY'] || ENV['http_proxy']
```

Comment:
I have mixed feelings about supporting `http_proxy` out of the box. My experience is some systems use `http_proxy` for HTTP proxies only and `https_proxy` for HTTPS, other systems use `http_proxy` for both. I think if we are going to support this `https_proxy` should dominate since these should always be HTTPS connections.